pydra/engine/tests/test_specs.py

```diff
@@ -27,7 +27,7 @@
 
 def test_basespec():
     spec = BaseSpec()
-    assert spec.hash == "06fe829a5dca34cc5f0710b454c24808"
+    assert spec.hash == "0b1d98df22ecd1733562711c205abca2"
 
 
 def test_runtime():
```

Comment:
I am a bit confused why the hash values should change. Aren't the test values in the specifications still the same as before?

Reply:
The test values are the same, just the type of the outer container and the index that `_graph_checksums` is inserted changes